src/robotto.js

```diff
@@ -82,4 +82,44 @@ robotto.parse = function(robotsFile) {
     return rulesObj;
 };
 
+robotto._stripRoute = function(urlParam) {
+    // Removes domain name and returns only the endPoint
+    return urlParam.replace(/^.*\/\/[^\/]+/, '');
+};
+
+robotto.check = function(userAgent, urlParam, rulesObj) {
+    let userAgents = Object.keys(rulesObj);
+    let desiredRoute = this._stripRoute(urlParam) + '/';
+    let allowed = true;
+
+    // Searches for every user agent until it gets a match
+    userAgents.forEach((agent) => {
+        if (agent.indexOf(userAgent) === 0) {
```

Comment:
this will be true for a partial match
why cant we just compare using === ?
```javascript
if (agent === userAgent) {
```
If it must be case insensitive then we can toLowerCase both of them